pkgs/by-name/la/lazynpm/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```

`finalAttrs` is usually preferred over `rec` because [it lends itself better to overriding](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)